packages/ember-data/lib/system/store.js

```diff
@@ -1377,11 +1391,12 @@ Store = Service.extend({
     adapter's `createRecord`, `updateRecord` or `deleteRecord`
     is rejected (with anything other than a `DS.InvalidError`).
 
-    @method recordWasError
+    @method _recordWasError
     @private
     @param {InternalModel} internalModel
   */
-  recordWasError: function(internalModel) {
+  _recordWasError: function(internalModel) {
+    Ember.deprecate('_recordWasError was documented as private and will be removed in the next version of Ember Data.');
```

Comment:
We want to keep the underscore method working, and create a new method called `recordWasError` that calls the previous one:

```javascript
recordWasError: function(internalModel) {
  Ember.deprecate('message');
  return this._recordWasError(internalModel);
}
```